src/main/scala/com/github/sbt/git/GitPlugin.scala

```diff
@@ -112,8 +112,8 @@ object SbtGit {
   // Build settings.
   import GitKeys._
   def buildSettings = Seq(
-    useConsoleForROGit := false,
-    gitReader := new DefaultReadableGit(baseDirectory.value, if (useConsoleForROGit.value) Some(new ConsoleGitReadableOnly(ConsoleGitRunner, file("."), sLog.value)) else None),
+    useConsoleForROGit := isGitLinkedRepo(baseDirectory.value),
+    gitReader := new DefaultReadableGit(baseDirectory.value, if (useConsoleForROGit.value) Some(new ConsoleGitReadableOnly(ConsoleGitRunner, baseDirectory.value, sLog.value)) else None),
```

Comment:
Likely not an oversight, but apparently this was made in an attempt to work with worktree as well:
https://github.com/sbt/sbt-git/commit/1fa3ca540cff9f39e96d763e04bdc8740467ac1a

Reply:
> It is probably not a real problem anyway since I doubt anyone would ever set it up like this. Do you agree?

I don't even know what a worktree is, so I'm like 🤷